Sources/SwiftDocC/Infrastructure/Workspace/DocumentationBundle+Info.swift

```diff
@@ -134,17 +146,25 @@ extension DocumentationBundle {
             }
             
             /// Helper function that decodes a value of the given type for the given key
-            /// in either the Codable container or Info.plist fallbacks.
+            /// in either the Codable container or Info.plist fallbacks or an optional fallback value.
```

Comment:
in either the Codable container, Info.plist fallbacks or an optional fallback value.